flink-core/src/main/java/org/apache/flink/configuration/RestartStrategyOptions.java

```diff
@@ -63,21 +63,21 @@
                                                     code("fixeddelay"),
                                                     code("fixed-delay"),
                                                     link(
-                                                            "{{.Site.BaseURL}}docs/ops/state/task_failure_recovery#fixed-delay-restart-strategy",
+                                                            "{{.Site.BaseURL}}/{{.Site.LanguagePrefix}}/docs/ops/state/task_failure_recovery#fixed-delay-restart-strategy",
```

Comment:
No other link seems to be using the LanguagePrefix, why should this one? Can we not use `{{< ref "docs/.." >}}` because the generated files are treated as short-codes?

Reply:
Thank you for reviewing, @zentol! `LanguagePrefix` is used here because we think it's better to let the links in the Chinese version direct to the page in the Chinese version, rather than the English version. With `LanguagePrefix` the link will be like `https://ci.apache.org/projects/flink/flink-docs-master/zh/docs/...`.

I've tried `{{< ref "docs/.." >}}`, but it doesn't works. The error message is: `unexpected "<" in command`. I'm not sure whether `ref` is available for the static html or not. 

Here's something I found that may be helpful: https://discourse.gohugo.io/t/using-shortcodes-in-html-content/2960/5